src/components/App.js

```diff
@@ -11,12 +11,20 @@ function App(props) {
       <div className="app">
         <Header />
 
-        <p className="warning">
-          The tBTC dApp is in ALPHA. Improper use can lead to LOSS OF FUNDS. If
-          you're uncomfortable with these risks, consider using another method
-          to mint TBTC. Visit <a href="https://discord.gg/Bpzfsgx">our
-          Discord</a> for options.
-        </p>
+        <div className="warning">
+          <div className="message">
```

Comment:
Do we a nested div here? It feels like we should be able to either use padding on `.warning` or a common left margin and max-width on `.warning p` to achieve the effects we're currently using `.message` for.

No need to block here, but it would be nice to take this in a follow-up.

Reply:
I had a feeling you might say exactly this 😆 😭. Will clean it up.

Reply:
Addressed in f981b18